twingly-http.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
   s.license     = "MIT"
   s.required_ruby_version = "~> 2.5"
 
-  s.add_dependency "faraday", "~> 1.0.1"
+  s.add_dependency "faraday", "~> 1.1.0"
```

Comment:
Could something like "~> 1" be an option too? Then we won't force updates if someone would happen to use 1.0.1 >= x < 1.1.0

Reply:
I'm not sure all new minor versions will still work for us, but if you are, we could change it :)

Reply:
~1, > 1.01 or whatver the syntax is ought to be at least as safe as before :)

Reply:
But then you allow, say, v1.9.0, right?

Reply:
Ah, now I get what you are getting at :) Since I think Faraday is using semver, it should be fine to support all 1.x versions

Reply:
I see. Then I'm buying this 😃

Reply:
Changed in 739f6bb.

Since there was no version between `1.0.1` and `1.1.0`, this change is equal to the one suggested in https://github.com/twingly/twingly-http/pull/12#discussion_r526755846

Reply:
Oops, i meant `>= 1.0.1`